rspace/src/test/scala/coop/rchain/rspace/nextgenrspace/history/HistoryRepositorySpec.scala

```diff
@@ -134,18 +133,23 @@ class HistoryRepositorySpec
     val emptyHistory =
       new History[Task](History.emptyRootHash, inMemHistoryStore, inMemPointerBlockStore)
     val pastRoots = rootRepository
-    pastRoots.commit(History.emptyRootHash).runSyncUnsafe(20.seconds)
 
-    val repo: TestHistoryRepository =
-      HistoryRepositoryImpl[Task, String, String, String, String](
+    (for {
+      _ <- pastRoots.commit(History.emptyRootHash)
+      repo = HistoryRepositoryImpl[Task, String, String, String, String](
         emptyHistory,
         pastRoots,
         inMemColdStore
       )
-    f(repo).runSyncUnsafe(20.seconds)
+      _ <- f(repo)
+    } yield ()).runSyncUnsafe(20.seconds)
   }
 }
 
+object RuntimeException {
+  def unapply(arg: RuntimeException): Option[String] = Some(arg.getMessage)
```

Comment:
`= Option(arg.getMessage)` should be used because it returns a `None` if `arg.getMessage == null`.
Isn't some util package a better place for this object so it can be re-used in other tests?